ui/js/dfv/src/fields/wysiwyg/index.js

```diff
@@ -41,6 +41,7 @@ const Wysiwyg = ( props ) => {
 		wysiwyg_editor: editor = 'tinymce',
 		wysiwyg_editor_height: editorHeight = 400,
 		wysiwyg_media_buttons: mediaButtons,
+		wysiwyg_wpautop: wpautop,
```

Comment:
```suggestion
		wysiwyg_wpautop: wpautop = true,
```

Reply:
It's off by default I think for new fields, so I just have that for now